### ISSUE TITLE
Metrics to determine model goodness. 

### ISSUE THREAD

**Issue by A143865** [reidsheppard/Python#2]
How do we determine how good our model is? 

- average difference of predicted number of yards vs actual number of yards?

As a benchmark can we use another model, like ESPN, and see how well that model predicts.

**Comment by sportygavin** [comment]
Yeah, my plan was to compare the percent error or difference of some sort between predicted and actual yards. I was then going to compare those from our model to ESPN's fantasy football model on a graph. 

- Our model vs ESPN's model (either predicted yards or percent error)
- Predicted yards vs actual yards

And then, look closer at the games where the model was more accurate and other games where it wasn't, and see if there is a reason for that within the games. Possible see if ESPN has different metrics that make their model better.

**Comment by A143865** [comment]
Have you coded up this metric calculation yet? I think you might have the start of it; would like to check if I missed something.

**Comment by sportygavin** [comment]
No, I haven't yet. All I have is the mean squared error and mean absolute error. I still have to find the percent error for each of the games and then do that for the ESPN model as well.
